docs/_integration-with-exisiting-apps-java.md

```diff
@@ -273,6 +288,8 @@ Perform a “Sync Project files with Gradle” operation.
 
 If you are using Android Studio, use `Alt + Enter` to add all missing imports in your MyReactActivity class. Be careful to use your package’s `BuildConfig` and not the one from the `facebook` package.
 
+> You need to run `yarn react-native run-android` to generate some classes. Even if this command fails the first time, it is needed to generate the necessary files to remove the Android Studio's error about the `PackageList` instantiation.
```

Comment:
I would move this inside the `### Enable native modules autolinking` section.

Reply:
@cortinico thanks for the comment, I've modified it